pkgs/by-name/en/encore/go.nix

```
@@ -0,0 +1,81 @@
+{ fetchFromGitHub, stdenv, lib, go, ... }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "go-encore";
+  version = "1.22";
+
+  src = fetchFromGitHub {
+    owner = "encoredev";
+    repo = "go";
+    rev = "encore-go${finalAttrs.version}";
+    hash = "sha256-SSa3CoUS6JMfs6T1PMb3NK8UeJnpLlTIn46/3oCTnL8=";
+  };
+
+  buildInputs = [ go ];
+
+  patchPhase = let
+    goSrc = fetchFromGitHub {
+      owner = "golang";
+      repo = "go";
+      rev = "release-branch.go${finalAttrs.version}";
```

The golang repo branches do not use the patch-number, so let's strip it (see https://github.com/golang/go/tree/release-branch.go1.22):
```suggestion
      rev = "release-branch.go${lib.versions.majorMinor finalAttrs.version}";
```